index.html

```diff
@@ -575,14 +573,14 @@ <h2>Syntax Tokens and Keywords</h2>
       <dt class="changed"><code>@nest</code></dt><dd class="changed">Collects a set of <a>nested properties</a> within
         a <a>node object</a>.</dd>
       <dt class="changed"><code>@none</code></dt><dd class="changed">Used as an index value
-        in an <a>index map</a>, <a>id map</a>, <a>language map</a>, <a>type map</a>, or elsewhere where a dictionary is
+        in an <a>index map</a>, <a>id map</a>, <a>language map</a>, <a>type map</a>, or elsewhere where a <a>map</a> is
         used to index into other values.</dd>
       <dt class="changed"><code>@prefix</code></dt><dd class="changed">
-        With the value <a>true</a>, allows this <a>term</a> to be used to construct a <a>compact IRI</a>
+        With the value <code>true</code>, allows this <a>term</a> to be used to construct a <a>compact IRI</a>
```

Comment:
A small little thing we added  to ReSpec is using back-ticks for simple `<code>` replacements,  like so:  

```suggestion
        With the value `true`, allows this <a>term</a> to be used to construct a <a>compact IRI</a>
```

Feel free to use that if you'd like.